app/src/androidTest/java/com/github/fribourgsdp/radio/mockimplementations/MockUserProfileActivity.kt

```diff
@@ -0,0 +1,28 @@
+package com.github.fribourgsdp.radio.mockimplementations
+
+import android.content.Context
+import android.os.Bundle
+import com.github.fribourgsdp.radio.*
+import org.mockito.Mockito
+import org.mockito.Mockito.mock
+
+const val userName = "test"
```

Comment:
Why don't you put this in the class directly?

Reply:
In the tests since the activity is initialized by an instance, i don't have access to that activites internals and I could not be bothered to make a companion object for a test but i can change it if you deem it is necessary

Reply:
No it is fine for me